src/js/tabs/firmware_flasher.js

```diff
@@ -1136,13 +1150,14 @@ firmware_flasher.initialize = function (callback) {
                         verifyBoard();
                     }
 
+                    updateDetectBoardButton();
```

Comment:
maybe the issue is here and we shoul tigered this at the end of function like before.

Reply:
Thanks could reproduce `Exit DFU Mode` not working - fixed.